tools/build_cros_gcc

```diff
@@ -142,37 +158,37 @@
  mkdir build-binutils || true
  cd build-binutils
  ../$BINUTILS/configure --prefix=$3 --target=$1 --disable-multilib
- make -j$(nproc)
+ make -j$(( `nproc` + 1 ))
```

Comment:
Why add 1?

Reply:
It makes the process go faster. Also recommended by Gentoo.

Reply:
Shouldn't this value be up to the user to decide?

Reply:
Not necessarily. I imagine most users would complain if this took days to compile. With this, it takes much less time.

Reply:
I know some builds require nproc = 1 or they fail.  I still think it should be up to the user to decide.

Reply:
Not any of these. That would be a major flaw, especially with GCC and Glibc.